src/components/button/button.component.tsx

```diff
@@ -269,6 +275,14 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
       [ref, forwardRef]
     );
 
+    const { marginBottom } = useContext<FormSpacingContextProps>(
+      FormSpacingContext
+    );
+    const marginProps = addFormSpacing(
+      filterStyledSystemMarginProps(rest),
+      marginBottom
+    );
+
```

Comment:
No you're right, I was talking to Harpal about it and wanted to demo what it would look like but I've then committed these changes despite agreeing they aren't needed 🤦